src/BookReader/PageContainer.js

```diff
@@ -70,6 +86,36 @@ export class PageContainer {
         });
     }
 
+    if (this.usePDF) {
+      this.drawPDFpage(reduce, dimensions);
+    }
+
     return this;
   }
+
+  async drawPDFpage(scale = 1, dimensions) {
+    console.log('****** drawPDFpage', this.index, this.pdf);
+    if (!this.index || !this.pdf?.getPage) {
+      return;
+    }
+    // Load information from the first page.
+    const page = await this.pdf?.getPage(this.index);
```

Comment:
Apparently pdfjs is 1-based.
```suggestion
    const page = await this.pdf?.getPage(this.index + 1);
```